Automattic/wp-calypso
# A4A: Implement the 'Hosting features' section on the new Migrations page.

This PR introduces the hosting feature section on the updated Migrations page.


<img width="1406" alt="Screenshot 2024-10-29 at 6 10 50 PM" src="https://github.com/user-attachments/assets/70f8910c-0c1e-48e2-ba1e-7de68a6081f8">

Depends on https://github.com/Automattic/wp-calypso/pull/95794
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1385

## Proposed Changes

* Implement the `MigrationsHostingFeatures` section component.

## Why are these changes being made?

* This implements the important section that showcases our hosting features.

## Testing Instructions


* Use the A4A live link and go to the `/migrations` page.
* Confirm you see the new Hosting features section.

## Pre-merge Checklist
